src/components/FollowButton/FollowButton.js

```diff
@@ -21,6 +21,16 @@ const propTypes = {
    */
   routeIdentifier: PropTypes.string.isRequired,
 
+  /**
+   * Button is active.
+   */
+  active: PropTypes.bool.isRequired,
+
+  /**
+   * Function triggered on button change.
+   */
+  onChange: PropTypes.func.isRequired,
```

Comment:
In fact, it's just a button. The function should be called onClick and then in the callback you change the value of active .

```
<FollowButton
  active={active}
  onClick={()=> {
     active=!active  
  });
/>
```